packages/use-intl/test/core/createFormatter.test.tsx

```diff
@@ -12,6 +12,18 @@ it('formats a date and time', () => {
 });
 
 it('formats a number', () => {
+  expect(intl.number(123456)).toBe(
+    '123,456'
+  );
+});
+
+it('formats a bigint', () => {
+  expect(intl.number(123456789123456n)).toBe(
```

Comment:
Is this a big int? I guess based on the suffix "n" it is, but the `Number.MAX_SAFE_INTEGER` is 9007199254740991, so this could be represented by a regular number as well.

Maybe we could make the number a bit larger so it exceeds `MAX_SAFE_INTEGER` just to be safe?

Reply:
Yep. `123n` and `BigInt(123)` (and `BigInt(123n)` for that matter) appear to be equivalent. Truly not sure how I managed to use a number smaller than `MAX_SAFE_INTEGER` but anyway I fixed it up with a larger number.